js/components/car-crash-instructions.js

```diff
@@ -0,0 +1,25 @@
+import React, { PureComponent } from 'react'
+import instructionStyles from '../../css/car-crash-instructions.less'
+
+const instructionImage = './common/images/crash/instructions.png'
+const instructionsText = `
+Use your mouse to drag the car on the left away from the parked car on the right.  Release the mouse and see how fast the dragged car is moving when it collides with the parked vehicle.
+
+Try starting the car at different locations. What do you notice?
+`
+export default class Instructions extends PureComponent {
```

Comment:
Seems like a good candidate for a functional component. You're using React 16.13, so hooks are available if you find yourself needing state or lifecycle access.

Reply:
Oops yes! Good point! Copy and paste, guilty as charged.  Thanks for pointing that out Kirk.